packages/gatsby/src/redux/types.ts

```diff
@@ -224,6 +224,11 @@ export interface IStaticQueryResultState {
   staticQueryResultHash: string
 }
 
+// @ts-ignore @typescript-eslint/no-empty-interface
+export interface IFeaturesState {
+  // imageService: boolean
```

Comment:
We could do some TS magic here with a `Features` enum mentioned above. Something along these lines could work...
```
export type IFeaturesState = {
    [feature: keyof typeof Feature]: boolean
}
```

That way we don't have to remember to update the state here any time we add a new feature. When a new feature is added to our enum, any state typehinting should make it instantly accessible.

Reply:
If only things were this simple ;) Our gatsby types are managed manually and they don't really flow nicely at this moment so  currently we need to duplicate these bits.